projects/pastanaga-angular/src/lib/chips/chip/chip.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { Observable } from 'rxjs';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+
+@Component({
+    selector: 'pa-chip',
+    templateUrl: './chip.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ChipComponent {
+    @Input() contact?: { name: string; image?: Observable<Blob> };
```

Comment:
split in two inputs: one for the text, one for the image.
I would replace the `ng-content` by the input text, so you can directly use the text passed as avatar's username.